borgmatic/commands/arguments.py

```diff
@@ -28,6 +28,37 @@
 }
 
 
+def parse_subparser_arguments(arguments, unparsed_arguments, subparsers):
```

Comment:
parse_subsubparser_arguments?

Reply:
It's not only doing subsubparsers.. This is also the code doing the actual parsing for each top-level subparser (and returning remaining arguments, of course). So maybe `parse_known_subparses_arguments()` or something?

Reply:
Another thought: Sometimes, if a function is tough to name, it means you haven't captured a coherent, describable unit of functionality with it. Like maybe it either does too much or too little such that it's hard to describe succinctly. So one thing you _could_ do in this case is to only make the function contain the "# Determine the remaining arguments that no subparsers have consumed" code and not the other loop. That way it'd be easier to create test cases for and it'd be easier to name. Just an idea.